shell/InstallToolchain.ps1

```diff
@@ -0,0 +1,4 @@
+if (Get-Command "choco.exe" -ErrorAction SilentlyContinue) 
+{ 
+    "7zip", "bitvise-ssh-server", "git", "notepadplusplus" | ForEach-Object -Process {choco install $_}
+} 
```

Comment:
No need for these to be here. They were pre-installed on the older system

Reply:
See https://github.com/chocolatey/chocolatey-test-environment/issues the tool chain is very old and I'd like to have the tools upgraded. 

We can fork the packer project and add the tools in packer but this keeps the design clean and installs current tools in this project. 

See my comments about using chocolatey to install the tools because I view the test environment as testing chocolatey packages and not chocolatey itself it's ok(?) to use chocolate to install the latest version of tools?

Reply:
> See my comments about using chocolatey to install the tools because I view the test environment as testing chocolatey packages and not chocolatey itself it's ok(?) to use chocolate to install the latest version of tools?

That's likely fine.